### DISCUSSION TITLE
How to install for the first time

### DISCUSSION THREAD

**Discussion by senbaph08** [oobabooga/text-generation-webui#4046] [Q&A] [answered]
I am trying to install Text generation web UI for the first time, but there is no download link for the installer in the One-click installers section of the README.
Where can I find the download link?

**Comment by sbuldan** [comment, marked as answer]
Download github, 
install it,
click the open in github desktop (or something like that)
specify the folder for installation and intall
goto the folder you have specified than click the start bat corresponding your operating system.
